TransIT.DAL/Models/DTOs/IssueLogDTO.cs

```diff
@@ -7,11 +7,13 @@ public class IssueLogDTO
         public int Id { get; set; }
         public string Description { get; set; }
         public decimal? Expenses { get; set; }
-        public int? OldStateId { get; set; }
-        public int? NewStateId { get; set; }
-        public int? SupplierId { get; set; }
-        public int? ActionTypeId { get; set; }
-        public int? IssueId { get; set; }
+        public StateDTO OldStateId { get; set; }
+        public StateDTO NewStateId { get; set; }
+        public SupplierDTO SupplierId { get; set; }
+        public ActionTypeDTO ActionTypeId { get; set; }
+        public IssueDTO IssueId { get; set; }
+        public int? CreateId { get; set; }
+        public int? ModId { get; set; }
```

Comment:
This issue was created by id = 5
Should front-end make one more call to retrieve user information (think about authorization, not everybody can gain details about other user)?
I suggest:
`public UserDTO CreatedBy`
`public UserDTO ModifiedBy`